projects/neural_correlations/EXP2-HighOrder/NeuCorr_Exp2.ipynb

```diff
@@ -77,13 +79,31 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "collapsed": true
    },
    "outputs": [],
    "source": [
+    "# create sequences\n",
     "allSequences = []\n",
+    "for s in range(numSequences):\n",
+    "    if s == 0:\n",
+    "        sequence = generateRandomSequence(symbolsPerSequence, tm.numberOfColumns(), sparsity)\n",
+    "    else:\n",
+    "        sequence = randomizeSequence(sequence, symbolsPerSequence, tm.numberOfColumns(), sparsity, 0.2)\n",
```

Comment:
This process generates "pseudo" high-order sequences. It just takes an initial sequence and substitutes 20% of its constituents (chosen at random) by random SDRs.